integrated_channels/sap_success_factors/models.py

```diff
@@ -142,7 +143,8 @@ def encrypted_key(self, value):
         blank=True,
         default='',
         verbose_name="Client Secret",
-        help_text=_("OAuth client secret.")
+        help_text=_("OAuth client secret."),
+        null=True
     )
```

Comment:
have we checked how these nulls are treated in DB when stored in encrypted form?

Reply:
@sameenfatima78 as the unencrypted columns are not in the model in django admin side also all other methods of updating these have been removed, record creation/updation is being done as expected.

the change is mainly required for tests when record is created through factory but no value is given for these unencrypted columns.